.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
 <!--
 Thank you for reporting an issue.
 
+*IMPORTANT* - please search our issue tracker *first* to ensure the
```

Comment:
We could always point them to our [contributing guidelines](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#reporting-bugs).